### PR TITLE
#613 added test for People

### PR DESCRIPTION
**Pull request by krzyk**
#613 
* added tests for People
* updated the coverage ratios
* added todo to continue increasing the coverage

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1027](https://github.com/zerocracy/farm/issues/1027) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1027](https://github.com/zerocracy/farm/issues/1027) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1381937017); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer

File: src/test/java/com/zerocracy/pmo/PeopleTest.java [hunk 1]
```diff
@@ -299,4 +380,33 @@ public void throwIfgetAppliedDateIfNotApplied() throws Exception {
...
+            "yegor%s", new Random().nextInt()
+        ).asString();
+        this.exception.expect(SoftException.class);
+        this.exception.expectMessage(
+            new FormattedText(" not valid: `%s`", wallet).asString()
+        );
+        people.wallet(uid, bank, wallet);
+    }
+
+    private void setsWallet(final String bank, final String wallet)
```
Comment (carlosmiranda) [inline src/test/java/com/zerocracy/pmo/PeopleTest.java]: @krzyk this can be static.

File: src/test/java/com/zerocracy/pmo/PeopleTest.java [hunk 2]
```diff
@@ -299,4 +380,33 @@ public void throwIfgetAppliedDateIfNotApplied() throws Exception {
...
+        throws IOException {
+        final People people = new People(new FkProject()).bootstrap();
+        final String uid = new FormattedText(
+            "yegor%s", new Random().nextInt()
+        ).asString();
+        people.wallet(uid, bank, wallet);
+        MatcherAssert.assertThat(
+            people.wallet(uid),
+            new IsEqual<>(wallet)
```
Comment (carlosmiranda) [inline src/test/java/com/zerocracy/pmo/PeopleTest.java]: @krzyk let's use `Matchers.is(wallet)` here.

File: src/test/java/com/zerocracy/pmo/PeopleTest.java [hunk 3]
```diff
@@ -105,6 +117,75 @@ public void setsUserWallet() throws Exception {
         );
     }
 
+    @Test
+    public void failsForIncorrectBankName() throws Exception {
+        final People people = new People(new FkProject()).bootstrap();
+        final String uid = new FormattedText(
+            "yegor%s", new Random().nextInt()
```
Comment (carlosmiranda) [inline src/test/java/com/zerocracy/pmo/PeopleTest.java]: @krzyk why do we need `Random` here, can't we use a fixed value?

File: src/test/java/com/zerocracy/pmo/PeopleTest.java [hunk 4]
```diff
@@ -299,4 +380,33 @@ public void throwIfgetAppliedDateIfNotApplied() throws Exception {
         people.invite(uid, uid);
         people.appliedTime(uid);
     }
+
+    private void failsWallet(final String bank)
+        throws IOException {
+        final String wallet =
+            new FormattedText("%d", new Random().nextInt()).asString();
```
Comment (carlosmiranda) [inline src/test/java/com/zerocracy/pmo/PeopleTest.java]: @krzyk same here, why do we need `Random`, can't we use a fixed value?

File: src/test/java/com/zerocracy/pmo/PeopleTest.java [hunk 5]
```diff
@@ -299,4 +380,33 @@ public void throwIfgetAppliedDateIfNotApplied() throws Exception {
...
+            new FormattedText(" not valid: `%s`", wallet).asString()
+        );
+        people.wallet(uid, bank, wallet);
+    }
+
+    private void setsWallet(final String bank, final String wallet)
+        throws IOException {
+        final People people = new People(new FkProject()).bootstrap();
+        final String uid = new FormattedText(
+            "yegor%s", new Random().nextInt()
```
Comment (carlosmiranda) [inline src/test/java/com/zerocracy/pmo/PeopleTest.java]: @krzyk same here, why do we need `Random`, can't we use a fixed value?

**Comment by krzyk** [comment]
@carlosmiranda updated

**Comment by krzyk** [comment]
@rultor merge

**Comment by 0crat** [comment]
@carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by yegor256** [comment]
@rultor merge

**Comment by krzyk** [comment]
@yegor256 updated, please merge again (this PR is very sensitive to changes in master)

**Comment by g4s8** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by krzyk** [comment]
@yegor256 please try to merge again (fixed pom formatting)

**Comment by krzyk** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1027](https://github.com/zerocracy/farm/issues/1027) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +20 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**